Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyScreen.kt

```diff
@@ -174,43 +177,47 @@ private fun TopAppBarTitle(
     )
 }
 
-@Suppress("DEPRECATION") // ConstraintLayout
-@OptIn(ExperimentalLayout::class)
+@Preview
+@Composable
+fun TAPPreview() {
```

Comment:
Is this name correct? If you're referring to TopAppBar, maybe it should be TABPreview instead? But I'd suggest writing it out anyway.

Reply:
It should be `Preview<NameOfComposable>` according to convention (I think there's a lint rule for this?)